assets/queries/dockerfile/copy_instead_of_add/query.rego

```diff
@@ -0,0 +1,23 @@
+package Cx
+
+CxPolicy [ result ] {
+  resource := input.document[i].command[j]
+  check(resource) 
+  not tarfileChecker(resource.Value, "tar")
```

Comment:
Maybe should be a little more specific and check with ".tar" and/or ".tar.", so that the extension is captured and not some filename containing tar, like "tarte.txt"